docs/local.md

```diff
@@ -15,88 +15,72 @@ if you want a more sophisticated setup.
+mode. "regtest" mode is stand-alone (non-networked) mode of monerod for testing purposes.
 
-Start monerod for regtest, this binary is in the monero bin directory:
-```bash
-cd ./monero-bin
-./monerod --regtest --fixed-difficulty=1 --rpc-bind-ip 127.0.0.1 --rpc-bind-port 18081 --offline
-```
+#### Build and run
 
-Create a wallet for "Bob", who will own XMR later on:
+Build binary:
 ```bash
-./monero-wallet-cli // you will be prompted to create a wallet. In the next steps, we will go with "Bob", without password. Remember the name and optionally the password for the upcoming steps
+make build
 ```
 
-You do not need to mine blocks, and you can exit the wallet-cli once Bob's account has been created by typing "exit".
+This creates `swapd` and `swapcli` binaries in the root directory.
 
-Start monero-wallet-rpc for Bob on port 18083. Make sure `--wallet-dir` corresponds to the directory the wallet from the previous step is in:
+To run as Alice, execute in terminal 1:
 ```bash
-./monero-wallet-rpc --rpc-bind-ip 127.0.0.1 --rpc-bind-port 18083 --password "" --disable-rpc-login --wallet-dir .
+./swapd --dev-xmrtaker
 ```
 
-Open the wallet:
+Alice will print out a libp2p node address, for example
+`/ip4/127.0.0.1/tcp/9933/p2p/12D3KooWAAxG7eTEHr2uBVw3BDMxYsxyqfKvj3qqqpRGtTfuzTuH`.
+This will be used for Bob to connect. You can either grab this address from the
+logs, our you can obtain it with this command:
 ```bash
-curl http://localhost:18083/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"open_wallet","params":{"filename":"Bob","password":""}}' -H 'Content-Type: application/json'
-
-# {
-#   "id": "0",
-#   "jsonrpc": "2.0",
-#   "result": {
-#   }
-# }
+./swapcli addresses
 ```
-
-Determine the address of `Bob` by looking at `monero-wallet-rpc` logs, in our case 45GcPCB ... uLkV5bTrZRe
+Pick the localhost address and assign it to a variable. For example (your value will be different):
 ```bash
-# 2022-01-20 21:40:06.460	W Loaded wallet keys file, with public address: 45GcPCBQgCG3tYcYqLdj4iQixpDZYw1MGew4PH1rthp9X2YrB2c2dty1r7SwhbCXw1RJMvfy8cW1UXyeESTAuLkV5bTrZRe
+BOOT_NODE=/ip4/127.0.0.1/tcp/9933/p2p/12D3KooWHRi24PVZ6TBnQJHdVyewDRcKFZtYV3qmB4KQo8iMyqik
 ```
-
-Then, mine some blocks on the monero test chain by running the following RPC command, replacing the address with the one from Bob's wallet:
+Now get the ethereum contract address that Alice deployed to. This can be pulled from the Alice's logs,
+the file ..., or if you have `jq` installed (available via `sudo apt install jq`), you can set a
+variable like this:
 ```bash
-curl -X POST http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"generateblocks","params":{"wallet_address":"45GcPCBQgCG3tYcYqLdj4iQixpDZYw1MGew4PH1rthp9X2YrB2c2dty1r7SwhbCXw1RJMvfy8cW1UXyeESTAuLkV5bTrZRe","amount_of_blocks":100}' -H 'Content-Type: application/json'
+CONTRACT_ADDR=$(jq -r .ContractAddress /tmp/xmrtaker/contract-address.json)
```

Comment:
This could be documented with `grep` and `sed`, but it is a little uglier. I suspect most developers have `jq` installed.  I'm using `jq` in `scripts/run-integration-tests.sh` for this same purpose.